gandalf.js

```diff
@@ -1,14 +1,14 @@
 /**
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?